### ISSUE TITLE
whiteboard content not getting updated in results.json file

### ISSUE THREAD

**Issue by marioaag** [avocado-framework/avocado#4717]
Using an avocado instrumented test with the nrunner ran a quick test to write in the whiteboard:
```
self.whiteboard = base64.encodebytes(b'My message encoded in base64').decode('ascii')
```

And found that the information is not replicated from the raw whiteboard file to the results.json file

```
{
    "cancel": 0,
    "debuglog": "/tmp/avocado_logs/job-2021-06-23T11.59-66cf07f/job.log",
    "errors": 0,
    "failures": 0,
    "interrupt": 0,
    "job_id": "66cf07fc7f9b4de85cfcbfee4479fb8701395877",
    "pass": 1,
    "skip": 0,
    "tests": [
        {
            "end": 0.90356296,
            "fail_reason": "<unknown>",
            "id": "230621-165951523698.BlueGenParallel.-1-/Users/marioalvarado/git/iLabConfig/test/avocado/testcases/CheckHostStatus/CheckHostStatus.py:CheckHostStatus.test_os_info",
            "logdir": "/tmp/avocado_logs/job-2021-06-23T11.59-66cf07f/test-results/230621-165951523698.BlueGenParallel.-1-_Users_marioalvarado_git_iLabConfig_test_avocado_testcases_CheckHostStatus_CheckHostStatus.py_CheckHostStatus.test_os_info",
            "logfile": "<unknown>",
            "start": 0.502914555,
            "status": "PASS",
            "tags": {},
            "time": 0.400648405,
            "whiteboard": "<unknown>"
        }
    ],
    "time": 0.400648405,
    "total": 1,
    "warn": 0
}
```

**Comment by clebergnu** [comment]
@marioaag this should be fixed with #4729, please reopen if you still find the problem.

**Comment by marioaag** [comment]
It works as expected!
